.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
```

Comment:
> - [ ] As you continue to perform a code review👍, it's important to provide feedback that is polite, constructive, and easy to understand. When giving feedback to the student, it's important to phrase the comments as suggestions rather than orders. For example, instead of saying "all Linters need to pass," you could say "have you considered checking the Linters to make sure they pass?"☺
> 
> - [ ] In addition, it's important to include an explanation or a "why" for each requested change. For example, instead of saying "you need to check your CSS fail," you could say "it would be helpful to review the CSS to ensure it's aligned with the project requirements."😊

Reply:
Kindly check if all your Linters? I noticed to you that Linters are your first filter that helps to get a clear code; Have you considered checking the Linters errors and fixing them? 🚗 

I'm sure when Linters will pass, you can understand that this tool can help us to be each day better! 📚 

It would be helpful to review the CSS to ensure it's aligned with project requirements! 🚋 🚋 🚋